scormxblock/static/html/scormxblock.html

```diff
@@ -1,31 +1,62 @@
 {% load i18n %}
-
+​
 <div class="scormxblock_block">
     {% if scorm_file_path %}
-
-    {% if scorm_xblock.has_score %}
-    <p>
-        (<span class="lesson_score">{{ scorm_xblock.lesson_score }}</span>/{{ scorm_xblock.weight }} {% trans "points" %})
-        <span class="completion_status">{% trans completion_status %}</span>
-    </p>
+        {% if scorm_xblock.open_in_pop_up %}
+            <!-- Trigger/Open The Modal -->
+            <button id="scormModalBtn">View Content</button>
+​
+            <!-- The Modal -->
+            <div id="scormModal" class="modal scorm-modal">
```

Comment:
nit: `id` of element needs to have xblock's block id as suffix otherwise we'd have if conflict if we place more than one scorm blocks inside single unit. May be we can get block id like [this](https://github.com/edly-io/xblock-video/blob/dev/video_xblock/mixins.py#L585).